### ISSUE TITLE
Yarn promptions quit without any outputs.

### ISSUE THREAD

**Issue by HakurouKen** [nathanhleung/install-peerdeps#70]
```
> npx install-peerdeps eslint-config-airbnb-base
install-peerdeps v2.0.1
It seems as if you are using Yarn. Would you like to use Yarn for the installation? (y/n) y
```

Error happened due to the next lines:

https://github.com/nathanhleung/install-peerdeps/blob/0db94a11a19f68a514ca148eead1e2a9a9cfeede/src/cli.js#L180-L194

`promptly.confirm` won't accept a callback as the second params in ^3.0.3 any more. 
See their docs in https://github.com/moxystudio/node-promptly#confirmmessage-options

**Comment by nathanhleung** [comment]
Should be closed by #66, thanks for waiting!

**Comment by nathanhleung** [comment]
Also, released in 2.0.2
